core/lib/web3_decl/src/namespaces/zks.rs

```diff
@@ -118,4 +118,8 @@ pub trait ZksNamespace {
         keys: Vec<H256>,
         l1_batch_number: L1BatchNumber,
     ) -> RpcResult<Proof>;
+
+    #[method(name = "getBatchPubdata")]
+    async fn get_batch_pubdata(&self, l1_batch_number: L1BatchNumber)
+        -> RpcResult<Option<Vec<u8>>>;
```

Comment:
Please use `zksync_type::Bytes` instead of `Vec<u8>` so that the returned value is hex-encoded rather than being a JSON array of `u8`s.